platform/linux-generic/pktio/subsystem.c

```diff
@@ -44,15 +54,23 @@ ODP_SUBSYSTEM_CONSTRUCTOR(pktio_ops)
 #ifdef ODP_PKTIO_DPDK
 	enable_link_dpdk_pktio_ops = 1;
 #endif
+#ifdef ODP_PKTIO_IPC
 	enable_link_ipc_pktio_ops = 1;
+#endif
 	enable_link_loopback_pktio_ops = 1;
```

Comment:
Any reason to keep loopback pkt IO enabled all the time?

Reply:
Right now, loopback pktio is built for both linux-generic and linux-dpdk (so I did not needed to disable it for one of the platforms). In the next step (see Yi's comment above), a pktio will be enable/disable at build time and all those flags will be removed.